.circleci/config.yml

```diff
@@ -111,4 +111,4 @@ workflows:
             - hmpps/deploy_env
           filters:
             branches:
-              only: master
+              only: main
```

Comment:
also rename `master` to `main`

Reply:
Not sure this will work.  I tried it once but the orb tools orb didn't support it, IIRC - https://circleci.com/developer/orbs/orb/circleci/orb-tools`

Reply:
it might do now though, just checking....

Reply:
looks like it might work.